rpm.if

```diff
@@ -431,9 +431,8 @@ interface(`rpm_named_filetrans',`
 	logging_log_named_filetrans($1, rpm_log_t, file, "yum.log")
 	logging_log_named_filetrans($1, rpm_log_t, file, "hawkey.log")
 	logging_log_named_filetrans($1, rpm_log_t, file, "up2date")
-	files_var_filetrans($1, rpm_var_lib_t, dir, "dnf")
-	files_var_filetrans($1, rpm_var_lib_t, dir, "yum")
-	files_var_filetrans($1, rpm_var_lib_t, dir, "rpm")
```

Comment:
Why file named transition for "rpm" directory is missing?

Reply:
Omitted intentionally as there rpm does not seem to use any other directory in /var than /var/lib/rpm which is confined, namely there is no /var/cache/rpm. The var_t type is assigned to /var and /var/.* when no other particular type is assigned.

Reply:
Understand, but it would be great to discuss it with RPM people to be sure. We want to avoid regressions.

Reply:
I raised the question. I doubt though as we don't have the support in rpm.fc.